### ISSUE TITLE
Image URL's have hard-coded context root WebGoat

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kipyegonmark/webgoat#15]
```
What steps will reproduce the problem?
1. Go to WebGoat front page
2. Go to useful tools under introduction

What is the expected output? What do you see instead?
I expect to see some screenshots, but no images are showing.

What version of the product are you using? On what operating system?
WebGoat 5.2 on Ubuntu 8.0.4

Please provide any additional information below.
To fix, code the URL with getResource() or other function that factors in
context root or use relative URL.  There are probably a lot more such
invalid img src's.
```

Original issue reported on code.google.com by `tse...@gmail.com` on 15 Jul 2008 at 4:08

**Comment by GoogleCodeExporter** [comment]
```
The img src URL should not start with "/WebGoat" but should instead start with
"/<context root>".  The images would have shown up if I have installed WebGoat 
at the
context root WebGoat, but instead I had it as WebGoat-5.2 (the default when I 
deploy
a WAR file).
```

Original comment by `tse...@gmail.com` on 15 Jul 2008 at 4:12

**Comment by GoogleCodeExporter** [comment]
```
Excellent point.  Will fix for 5.3
```

Original comment by `mayhe...@gmail.com` on 15 Jul 2008 at 11:18
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `mayhe...@gmail.com` on 21 Nov 2008 at 4:56
- Changed state: **Fixed**
